src/Microsoft.Bcl.HashCode/src/Microsoft.Bcl.HashCode.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+      <StrongNameKeyId>Open</StrongNameKeyId>
+      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+      <Nullable>enable</Nullable>
+      <AssemblyVersion>1.0.1.0</AssemblyVersion>

```

Comment:
Is this the right assembly version to use? According to https://github.com/dotnet/corefx/blob/b86ce69c316b4b088356b89bab80f7fa79ff799c/pkg/Microsoft.Private.PackageBaseline/packageIndex.json#L36-L48 , and by attempting to imitate what was done by @joperezr in System.Buffers, The latest version available in `AssemblyVersionInPackageVersion` is 1.0.1.0.